ansible/roles/ocp4-workload-workshop-admin-storage/tasks/pre_workload.yml

```diff
@@ -1,6 +1,41 @@
 ---
 # Implement your Pre Workload deployment tasks here
 
+- name: Cleaning admin
+  include_tasks:
+    file: ./clean-admin.yml
+
+- name: extract the sandbox id
+  shell: "echo {{ target_host }} | cut -d. -f3"
+  register: sandbox_id
+
+- name: extract api_url
+  command: oc whoami --show-server
+  register: api_url
+
+- name: extract master_url
+  shell: echo https://$(oc get route console -o jsonpath='{.spec.host}' -n openshift-console)
+  register: master_url
+
+- name: extract route_subdomain
+  command: oc get ingresses.config.openshift.io cluster -o jsonpath='{.spec.domain}'
+  register: route_subdomain
+
+- name: extract guid
+  shell: echo $(oc whoami --show-server | awk -F '[-.]' '{ print $3 }')
+  register: guid
```

Comment:
Do not use the `guid` variable.
This variable is *always* passed as extra vars and will override your register here.

You don't need to extract it.